test/unit/math/mix/scal/fun/F32_test.cpp

```diff
@@ -1,4 +1,7 @@
 #include <stan/math/mix/scal.hpp>
+#include <stan/math/mix/mat.hpp>
```

Comment:
Okay. Looking through the code, I think we could include `test/unit/math/rev/mat/fun/util.hpp` instead because the test only looks to be using AVEC.